lms/djangoapps/learner_dashboard/api/v0/tests/test_views.py

```diff
@@ -116,3 +127,84 @@ def test_404_if_no_program_data(self, mock_get_programs, _mock_get_pathways):
         response = self.client.get(self.url)
         assert response.status_code == 404
         assert response.data['error_code'] == 'No program data available.'
+
+
+class TestProgramsView(SharedModuleStoreTestCase, ProgramCacheMixin):
+    """Unit tests for the program details page."""
+
+    enterprise_uuid = str(uuid4())
+    program_uuid = str(uuid4())
+    password = 'test'
+    url = reverse_lazy('learner_dashboard:v0:program_list', kwargs={'enterprise_uuid': enterprise_uuid})
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.user = UserFactory(is_staff=True)
```

Comment:
Do we need this `is_staff=True`? Isn't this endpoint supposed to be accessed by any enterprise user?

Reply:
good catch. fixed now.